mava/systems/ippo/networks.py

```diff
@@ -170,10 +170,11 @@ class ValueHead(hk.Module):
     def __init__(
         self,
         name: Optional[str] = None,
+        w_init: Optional[hk.initializers.Initializer] = None,
```

Comment:
If you can add a docstring to define the attributes of the class:
w_init: Optional initializer for weights.
Also why only w_init can't we extend to have b_init too? or it is uncessary?

Reply:
Updated the docstring :) 
Please see my comment [here](https://github.com/instadeepai/Mava/pull/781#discussion_r989696451) about only having `w_init` in the `CategoricalValueHead` and `ValueHead` classes.

Reply:
I see thanks  for the explanation!